ec2-static-alarms/alarms.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_cloudwatch_metric_alarm" "alarms" {
+  for_each            = var.alarms
+  alarm_name          = each.key
+  comparison_operator = each.value.comparison_operator
+  metric_name         = each.value.metric_name
+  threshold           = each.value.threshold
+  namespace           = "AWS/EC2"
+  statistic           = "Average"
+  period              = 60 # seconds, less than this might cause error depending on EC2 configuration
+  evaluation_periods  = "1"
```

Comment:
for `period` and `evaluation_periods` - can we use as variable with default value

Reply:
Yes I will make it configurable.